### ISSUE TITLE
Making model personalised

### ISSUE THREAD

**Issue by ra-MANUJ-an** [facebookresearch/llama#835]
I came across this [post](https://www.reddit.com/r/LocalLLaMA/comments/13j3747/tutorial_a_simple_way_to_get_rid_of_as_an_ai/) by [u/Shir_man](https://www.reddit.com/u/Shir_man/) where it has been described as how to get rid of "..as an AI language model..." and making the model more personalised. This method needs each and every token id to be known in prior. Is there any way this issue can be circumvented and we don't require any token ids, but instead change the logits as done by [GeDi](https://github.com/salesforce/GeDi)? I'm not looking for fine-tuning based methods.

**Comment by melanierk** [comment]
Hi @ra-MANUJ-an, first, please make sure you are using the model according to the [acceptable use policy](https://github.com/facebookresearch/llama/blob/main/USE_POLICY.md) and the [responsible use guide](https://ai.meta.com/static-resource/responsible-use-guide/).


We haven't experimented with logit changing, but it may be possible. You could also try tinkering with the system prompt.
